hash_practice/exercises.py

```diff
@@ -2,18 +2,43 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O (n^2(logn))because I'm using the sorted function inside the loop 

```

Comment:
I get what you mean here, but be careful about reusing the same variable for complexity calculations. In this case you seem to be using `n` as both the number of strings in `strings` _and_ the number of letters in the words.  I would probably use something like `m` for the number of letters in the words to get O(n * m * log(m)).

However, given the limitation that the words are English words, we don't have to necessarily account for them since English words tend not to get very long. On average there are 4.7 letters in an English word, which is greatly dwarfed by the number of strings in `strings`, since there could be hundreds or thousands of strings in `strings`. As such, we might just consider this to be O(n) and disregard `m`.